config.yaml

```diff
@@ -1,6 +1,7 @@
 default:
  branch: 'powerkvm-v3.1.1'
  work_dir: 'build'
+ result_dir: 'result'
```

Comment:
Wouldn't it be better to allow different directories for package building (eg result/packages) and ISO building (eg result/iso) and not support this argument for other subcommands?